### ISSUE TITLE
how to remove request header cache control

### ISSUE THREAD

**Issue by 369857519** [FineUploader/fine-uploader#1726]
XMLHttpRequest cannot load http://1511655198.cloud.vimeo.com/upload?ticket_id=69d.......ae231c. Request header field Cache-Control is not allowed by Access-Control-Allow-Headers.

my CDN vender only support these ways

Access-Control-Allow-Headers:X-File-Name,X-File-Type,X-File-Size,key,Content-Type,Authorization,UploadBatch, x-requested-with

what should I do with fine-uploader?

**Comment by 369857519** [comment]
I did something like this

var uploader = new FineUploaderTraditional({
		   options: {
		      request: {
		         endpoint: 'http://xiaomiup.up9.v1.wcsapi.com/file/upload',
		         token:this.props.token,
		         customHeaders:{
		         	'Access-Control-Request-Headers':'x-requested-with'
		         }
		      }
		   }
		})

but it's not working

**Comment by egucciar** [comment]
why was this issue closed without a resolution? Why cant we remove the cache control header from the request using an option? @rnicholus  I dont want to fork your repo just to do that.

**Comment by rnicholus** [comment]
>Why cant we remove the cache control header from the request using an option?

Because there is no such option...yet.

**Comment by egucciar** [comment]
Thanks for the quick response. I ended up forking and modifying the distribution and publishing to npm with our prefix to work around this critical issue, because I can't control what the server accepts. But this solution is bad because I modified the dist (I couldn't get the build to work) and now I'm formed from your repo. If I have time this week or next I'll see about making an option in the source, but not totally optimistic because I couldn't even run the build.
